### DISCUSSION TITLE
"Move Node Left" in Outline View captures siblings beneath - intentional?

### DISCUSSION THREAD

**Discussion by JJSlote** [freeplane/freeplane#1396] [Help] [answered]
Greetings  

If we're in Outline view and "promote" a node one level higher via Ctrl+Left, the promoted node becomes _parent_ to its former siblings beneath. 

But that's not the behavior in an ordinary map view, where, when the branch is aligned rightward,  Ctrl+Left promotes the selected node without changing the parentage of other nodes. 

Is that difference in behavior intentional? I work in Outline View for compactness but much prefer the map view behavior on promotion. When adding nodes spontaneously as thoughts occur, they're typically at a single level. The ones I then promote as header candidates are not yet logically associated with the nodes beneath them. In a large map, I sometimes discover I've inadvertently hidden and demoted a set of siblings through parentage-by-promotion.

I do recognize that Ctrl+Left is designed to move nodes left, not to promote them per se; when branches are aligned to the left, for example, that key sequence will demote the nodes. But the behavior in Outline view has me looking for a workaround.

Is there a command I might attach to a keyboard sequence that simply promotes selected nodes, irrespective of branch alignment or outline view, without altering the parentage of unselected siblings?

Thanks for any thoughts. Demo map attached. 

Rgds - Jerome

-------
Version: 1.11.6 Windows
[DemoMoveNodeLeft.zip](https://github.com/freeplane/freeplane/files/12568374/DemoMoveNodeLeft.zip)

**Comment by euu2021** [comment, marked as answer]
Now, we have layouts that bring the compactness of an outline view to the map view.  Try, for example, this layout:

![image](https://github.com/freeplane/freeplane/assets/77707706/6207a747-b214-4dbe-9a2b-d960a8c33d51)

But, I do agree that the behavior that you mention sounds weird. I just tested Dynalist, and it doesn't work like that.

**Reply by JJSlote** [reply]
Thanks, euu2021. Scrivener also promotes nodes on that key sequence without turning the siblings beneath into descendants. But that behavior has the possible effect of reordering the nodes, so there's clearly a tradeoff. I can see where Outline View's "Move Node Left" is the purest, but the user must understand the effect on adjacent nodes when running that command on a large map. 

The outline-like layout you've suggested looks terrific, adds other styling advantages, and works exactly as we'd expect. Thanks again!
